modules/abstract-eth/src/ethLikeToken.ts

```diff
@@ -35,10 +36,6 @@ export class EthLikeToken extends AbstractEthLikeCoin {
     return tokensCtors;
   }
 
-  protected getTransactionBuilder(): EthTransactionBuilder {
-    return new EthTransactionBuilder(coins.get(this.getBaseChain()));
-  }
-
```

Comment:
Modified this method because TransactionBuilder in abstract-eth is an abstract class and we can't instantiate the abstract class